libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -69,6 +69,14 @@ object DataType {
     def isSuccess:Boolean = resultCode == 0 || resultCode == 1
   }
 
+  final case class InstallationTarget(target: TargetFilename,
+                                      hashes: Map[HashMethod, Refined[String, ValidChecksum]],
```

Comment:
do we need hash and size? I mean what is a chance they are reported from a device?